install.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+require('shelljs/global');
+var gup = require('guppy-cli');
+var async = require('async');
+var pack = require(rootApplicationPath() + '/package.json');
+
+// Install each confgured hook.
+pack['guppy-hooks'] && async.each(pack['guppy-hooks'], function (hook, next) {
```

Comment:
I would prefer it if you use an `if` here instead. That would make it more clear what's happening on this line and easier to maintain.